templates/index.html

```diff
@@ -35,4 +35,28 @@ <h2 class="p-heading--three"><a href="{{ featured_post.relative_link }}">{{ feat
 
   {% include "posts.html" %}
 
+<div class="p-strip is-shallow is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Webinars</h2>
+    {% for webinar in webinars %}
+      <div class="p-media-object" itemscope itemtype="http://schema.org/Event">
+        <div class="p-media-object__details">
+          <h3 class="p-media-object__title" itemprop="name">
+            <a href="{{ webinar.link | safe }}">{{ webinar.title | safe }}</a>
+          </h3>
+          <p class="p-media-object__content">{{ webinar.summary | safe }}</p>
+          <ul class="p-media-object__meta-list">
+            <li class="p-media-object__meta-list-item">
+              <span class="u-off-screen">Topic: </span>{{ webinar.category | safe }}
```

Comment:
I think as this has no image it would be worth displaying `Topic:`.

Reply:
Done